### PR TITLE
Fixed content view size to be correct

### PR DESCRIPTION
**Pull request by itsol-ka**
Content view size is incorrect when using autolayout for content view.
Fixed content view size to be correct.

maybe related #259

### CODE REVIEW

File: Tests/SurfaceViewTests.swift [hunk 1]
```diff
@@ -117,6 +117,27 @@ class SurfaceViewTests: XCTestCase {
         }
     }
 
+    func test_surfaceView_size_equalTo_containerView_size() {
+        let surface = SurfaceView(frame: CGRect(x: 0.0, y: 0.0, width: 320.0, height: 480.0))
+        let contentView = UIView()
+        let titleLabel = UILabel()
+        contentView.addSubview(titleLabel)
+        contentView.translatesAutoresizingMaskIntoConstraints = false
```
Comment (scenee) [inline Tests/SurfaceViewTests.swift:125]: This line should be `titleLabel.translatesAutoresizingMaskIntoConstraints = false`. And if it changes, then this test case is passed even if the priority is still `.required - 1`. So this test case is not a sufficient reason for the priority change.

The background where the priority was set to `.required - 1(= 999)` is https://github.com/scenee/FloatingPanel/pull/359.  So we have to take care of its case.

**Review by scenee — CHANGES_REQUESTED** [review]

File: Tests/SurfaceViewTests.swift (hunk 1 above)
Comment (itsol-ka) [reply Tests/SurfaceViewTests.swift:125]: Thank you for pointing out the test mistake.
I will rethink

**Comment by Batvos** [comment]
Hi,

This PR fixed my issue with content size when I use uicollectionview with animated snapshot inside content view.
@scenee Could merge it or clarify what should be done to accept this PR.

**Comment by scenee** [comment]
@Batvos Is the content mode(`FloatingPanelController.contentMode`) `static`? I think we need these priority change according the content mode.

**Comment by scenee** [comment]
Hi there,
I've finally acknowledged this priority issue to be resolved for the `.static` content mode in #515. So I've create #519.
I closed this PR but I really appreciate your PR to let me know this issue for the first time.

**Comment by itsol-ka** [comment]
Thank you for fixing the bug.

**Pull request closed without merging**